### PR TITLE
Feature/transmit exit code

### PR DESCRIPTION
**Pull request by kagux**

### CODE REVIEW

File: server/server.go [hunk 1]
```diff
@@ -54,9 +54,18 @@ func (s *Server) handleRequest(conn net.Conn) {
 		fmt.Println("Error reading:", err.Error())
```
Comment (juanperi) [inline server/server.go]: if there is an error, shouldn't be reported back as error to the client?

File: command/output.go [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+import (
+	"fmt"
+)
+
+type Output struct {
+	Text       string
+	ExitStatus int
+}
+
+func NewErrorOutput(err error) *Output {
```
Comment (juanperi) [inline command/output.go]: why the lack of symmetry? Write receives the outputWriter and writes to it. But this one returns the Output, and lets the caller to do the write

**Comment by juanperi** [comment]
other than the comments, +1

File: server/server.go (hunk 1 above)
Comment (kagux) [inline server/server.go]: done

File: command/output.go (hunk 2 above)
Comment (kagux) [inline command/output.go]: done

**Pull request merged**